src/ApplicationInsights.Kubernetes/IK8sQueryClient.cs

```diff
@@ -1,19 +1,16 @@
-﻿#nullable enable
-
-using System;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.ApplicationInsights.Kubernetes.Entities;
+using k8s.Models;
 
 namespace Microsoft.ApplicationInsights.Kubernetes
 {
-    internal interface IK8sQueryClient : IDisposable
+    internal interface IK8sQueryClient
```

Comment:
How much value is this interface provide? Is this the same as IK8sClientService? Why do we need this in the middle?